server/src/test/java/org/elasticsearch/index/engine/InternalEngineTests.java

```diff
@@ -1310,7 +1360,8 @@ public void testSyncedFlushSurvivesEngineRestart() throws IOException {
     }
 
     public void testSyncedFlushVanishesOnReplay() throws IOException {
-        IOUtils.close(store, engine);
+        store.close();
+        engine.close();
```

Comment:
I notice there's lots of places where we've gone from `IOUtils.close(engine, ...)` to `engine.close()`. So the null-safety is gone. I assume that's all intentional/safe?